File: jl/Practice2/Mathematical/GreedyTakahashi.jl
# https://atcoder.jp/contests/abc149/tasks/abc149_b

(
  readline() |> split |> a -> parse.(Int64, a) |> a -> (a[1], a[2], a[3])
  |> ((a, b, k),) -> (
    "$(max(0, a - k)) $(max(0, b - max(0, (k - a))))";
  )
  |> println
)

# https://atcoder.jp/contests/abc149/submissions/29733995

File: js/Practice2/Mathematical/GreedyTakahashi.js
// https://atcoder.jp/contests/abc149/tasks/abc149_b

const main = ([a, b, k]) =>
  `${Math.max(0, a - k)} ${Math.max(0, b - Math.max(0, (k - a)))}`;

console.log(main(require('fs').readFileSync(0).toString().split(' ')));

// https://atcoder.jp/contests/abc149/submissions/29732926
